pyiron_gpl/pacemaker/pacemaker.py

```diff
@@ -0,0 +1,231 @@
+# coding: utf-8
+# Copyright (c) ICAMS, Ruhr University Bochum, 2021
+# Distributed under the terms of "GPLv3", see the LICENSE file.
+
+import logging
+import numpy as np
+import os
+import pandas as pd
+import re
+import ruamel.yaml as yaml
+from pyiron_base import GenericJob, GenericParameters
+from pyiron_base.settings.generic import Settings
+from shutil import copyfile
+
+
+s = Settings()
+
+try:
+    from pyace import BBasisConfiguration, ACEBBasisSet
+
+    HAS_PYACE = True
+except ImportError as e:
+    print("Could not import `pyace` package. The package should be installed for proper functionality")
+    HAS_PYACE = False
+
+# set loggers
+loggers = [logging.getLogger(name) for name in logging.root.manager.loggerDict]
+for logger in loggers:
+    logger.setLevel(logging.WARNING)
+
+class PaceMakerJob(GenericJob):
+    def __init__(self, project, job_name):
+        super().__init__(project, job_name)
+        self.__name__ = "PaceMakerJob"
+        self.__version__ = "0.1"
+
+        self.input = GenericParameters(table_name="input")
```

Comment:
Please use `InputList` instead. The only immediate change I foresee is that it facilitates `.key` functionality instead of `['key']`, but it also helps future-proof the class against the day when we stop supporting `GenericParameters`.

Reply:
Done